docs/website/docs/general-usage/http/rest-client.md

```diff
@@ -191,7 +191,7 @@ E.g. `https://api.example.com/items?offset=0&limit=100`, `https://api.example.co
 
 ```json
 {
-  "items": [...],
+  "items": ["one", "two", "three"],
```

Comment:
Is there a way to leave ellipsis?

Reply:
it does not pass json parsing that way.. we can mark the block as text (which will disabled syntax highlighting) or write something like [{}, {}, {}].

Reply:
Let's use the strings then